src/pages/markets/components/helper/_symbol.tsx

```diff
@@ -11,14 +11,14 @@ const Symbol = ({ instruments_type, src, text }: SymbolProps) => (
             <React.Fragment>
                 {instruments_type.map((symbol, index) => (
                     <SymbolContainer key={index}>
-                        <img src={symbol.src} />
+                        <img src={symbol.src} alt="symbol" />
```

Comment:
`symbol.text`?

Reply:
I talked with u about these, we can't add text as a alt message and if we want to add alt message to all of the arrays it takes too much time